members/FMOS63398500.yaml

```diff
@@ -5,118 +5,121 @@ contact_form:
-        selector: "#edit-address-1"
-        value: $ADDRESS_STREET
-        required: true
-      - name: Street2
-        selector: "#edit-address-2"
-        value: $ADDRESS_STREET_2
-        required: false
-      - name: City
-        selector: "#edit-city"
-        value: $ADDRESS_CITY
-        required: true
-      - name: zip
-        selector: "#edit-zip"
-        value: $ADDRESS_ZIP5
-        required: true
-      - name: phone
-        selector: "#edit-phone"
-        value: $PHONE
-        required: false
-      - name: email
-        selector: "#edit-email"
-        value: $EMAIL
-        required: true
-      - name: Subject
-        selector: "#edit-subject"
-        value: $SUBJECT
-        required: true
-      - name: Message
-        selector: "#edit-message"
-        value: $MESSAGE
-        required: true
+        - name: First name
+          selector: "#edit-first-name"
+          value: $NAME_FIRST
+          required: true
+        - name: Last name
+          selector: "#edit-last-name"
+          value: $NAME_LAST
+          required: true
+        - name: Street
+          selector: "#edit-address-1"
+          value: $ADDRESS_STREET
+          required: true
+        - name: Street2
+          selector: "#edit-address-2"
+          value: $ADDRESS_STREET_2
+          required: false
+        - name: City
+          selector: "#edit-city"
+          value: $ADDRESS_CITY
+          required: true
+        - name: zip
+          selector: "#edit-zip"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: phone
+          selector: "#edit-phone"
+          value: $PHONE
+          required: false
+        - name: email
+          selector: "#edit-email"
+          value: $EMAIL
+          required: true
+        - name: Subject
+          selector: "#edit-subject"
+          value: $SUBJECT
+          required: true
+        - name: Message
+          selector: "#edit-message"
+          value: $MESSAGE
+          required: true
     - javascript:
-      - name: Message
-        selectors: [ "#edit-message" ]
-        values: [ "$MESSAGE" ]
-        commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/\\b(set forth in the bill)\\b/ig,'').replace(/(%2[0C])/g, '').replace(/(;)/g,'').replace(/(—)/g,'').replace(/(%2d)/g,'');}"]
-        required: true
+        - name: Message
+          selectors: ["#edit-message"]
+          values: ["$MESSAGE"]
+          commands:
+            [
+              "for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/\\b(set forth in the bill)\\b/ig,'').replace(/(%2[0C])/g, '').replace(/(;)/g,'').replace(/(—)/g,'').replace(/(%2d)/g,'');}",
+            ]
+          required: true
     - click_on:
-      - value: Submit
-        selector: button#edit-actions-submit
+        - value: Submit
+          selector: button#edit-actions-submit
     - wait:
-      - value: 5
+        - value: 10
```

Comment:
Per https://ngpvan.atlassian.net/wiki/spaces/DIGITAL/pages/129269829/Advocacy+Deliverability+Troubleshooting#AdvocacyDeliverabilityTroubleshooting-Problem%3AErrorscreenshotindicatesthataTopicwasnotselectedonthetarget'scontactform - looks like past recommendation was to add a "find" instead of a "wait".

Reply:
@rebeccacsanders made requested changes